### PR TITLE
some jiras from the events section stu2

### PR DESCRIPTION
**Pull request by isaacvetter**
FHIR-25655
FHIR-25838
FHIR-25840
FHIR-25841
FHIR-25843
FHIR-25846
FHIR-25852
FHIR-25853

### CODE REVIEW

**Comment by isaacvetter** [comment]
Either Niklas approves #316, then I merge this into that, or you approve this one and we abandon #316 ...

File: docs/specification/STU2.md [hunk 1]
```diff
@@ -424,27 +426,28 @@ Authorization: Bearer i8hweunweunweofiwweoijewiwe
...
+		"context": [{
+			"key": "operationoutcome",
+			"resource": {
+				"resourceType": "OperationOutcome",
+				"issue": [{
+					"severity": "warning",
+					"code": "processing",
+					"diagnostics": "AppId3456 failed to follow context"
+				}],
+				"details": {
```
Comment (wmaethner) [inline docs/specification/STU2.md:443]: Based on the FHIR site it looks like the details is also part of the issue property.
[https://www.hl7.org/fhir/operationoutcome.html](https://www.hl7.org/fhir/operationoutcome.html)

Comment (isaacvetter) [reply docs/specification/STU2.md]: 👍 great catch, thank you, sir! Fixed.

File: docs/specification/STU2.md [hunk 2]
```diff
@@ -400,16 +402,16 @@ If the subscriber cannot follow the context of the event, for instance due to an
...
+`id` | Required | *string* | Event identifier, which MAY be used to recognize retried notifications. This id SHALL be unique and could be a UUID. 
 `event` | Required | *object* | A json object describing the event. See [below](#event-notification-error-event-object-parameters).
 
 ###### Event Notification Error Event Object Parameters
 Field | Optionality | Type | Description
 --- | --- | --- | ---
 `hub.topic` | Required | string | The session topic given in the subscription request. 
 `hub.event`| Required | string | Shall be the string `syncerror`.
-`context` | Required | array | An array containing a single FHIR OperationOutcome. The OperationOutcome SHALL use a code of `processing`. 
+`context` | Required | array | An array containing a single FHIR OperationOutcome. The OperationOutcome SHALL use a code of `processing`. The OperationOutcome's details SHALL  contain the id of the event that this error is related to.
```
Comment (isaacvetter) [inline docs/specification/STU2.md:418]:
```suggestion
`context` | Required | array | An array containing a single FHIR OperationOutcome. The OperationOutcome SHALL use a code of `processing`. The OperationOutcome's details SHALL contain the id of the event that this error is related to. 
```

**Comment by isaacvetter** [comment]
@wmaethner - ready for your re-review!

**Pull request merged**